src/pretix/control/forms/event.py

```diff
@@ -274,6 +289,11 @@ def clean(self):
             raise ValidationError({
                 'attendee_names_required': _('You cannot require specifying attendee names if you do not ask for them.')
             })
+        if data['attendee_emails_required'] and not data['attendee_emails_asked']:
+            raise ValidationError({
+                'attendee_emails_required': _('You cannot require specifying attendee emails if you do not ask for '
```

Comment:
"You have to ask for attendee emails if you want to make them required." or something like that?